spec/helpers/search_helper_spec.rb

```diff
@@ -139,4 +141,14 @@
       expect(asc_item).to be == @audited_items[index]
     end
   end
+
+  it "puts calculates the average lend time correctly" do
+    calc_avg_lend_time = helper.statistics_item_lend_time(@audited_items[0])
+    expect(calc_avg_lend_time).to eq(1.day)
+  end
+
+  it "puts calculates the average lend time of a never borrowed before item correctly" do
+    calc_avg_lend_time = helper.statistics_item_lend_time(@item_book)
+    expect(calc_avg_lend_time).to eq(0.seconds)
+  end
```

Comment:
are these "puts" here intended?

Reply:
no thank you :) I'll remove it. Good catch ;)